plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py

```diff
@@ -957,17 +968,17 @@ def main():
             agent=dict(type='str'),
             email=dict(type='str'),
             description=dict(type='str'),
-            thresholds=dict(type='list', default=[]),
-            ports=dict(type='list', default=[]),
-            processes=dict(type='list', default=[]),
-            add_ports=dict(type='list', default=[]),
-            update_ports=dict(type='list', default=[]),
-            remove_ports=dict(type='list', default=[]),
-            add_processes=dict(type='list', default=[]),
-            update_processes=dict(type='list', default=[]),
-            remove_processes=dict(type='list', default=[]),
-            add_servers=dict(type='list', default=[]),
-            remove_servers=dict(type='list', default=[]),
+            thresholds=dict(type='list', elements="dict", default=[]),
+            ports=dict(type='list', elements="dict", default=[]),
+            processes=dict(type='list', elements="dict", default=[]),
+            add_ports=dict(type='list', elements="dict", default=[]),
+            update_ports=dict(type='list', elements="dict", default=[]),
+            remove_ports=dict(type='list', elements="str", default=[]),
+            add_processes=dict(type='list', elements="dict", default=[]),
+            update_processes=dict(type='list', elements="dict", default=[]),
+            remove_processes=dict(type='list', elements="str", default=[]),
+            add_servers=dict(type='list', elements="dict", default=[]),
```

Comment:
```suggestion
            add_servers=dict(type='list', elements="str", default=[]),
```
I *think* this should be str: https://github.com/ansible-collections/community.general/blob/0de6330d5c6ede5aea6987910d00df934323283c/plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py#L412-L413

Reply:
we should add @russoz as a mainainer of **all** the modules, so he will know about possible things introduced:D
(i'm joking)

Reply:
ahahahaha the same we do every night, Pinky: try and take over the world! :-)

yes, that one definitely seems like a `str` that escaped my attention